models/cam-se/model_mod.f90

```diff
@@ -1,3 +1,6 @@
+! "Debug pressure calc" shows changes which prevent users from asking for total(moist) pressure
+! without providing the moisture variables in the state vector.
+
```

Comment:
What is this comment referring to?  
It looks like some notes from the previous commit. I'd remove this or if it is a necessary comment, put the comment by the code it is describing.